Sample-App/app/build.gradle

```diff
@@ -45,4 +49,15 @@ dependencies {
     // Image loading
     implementation 'com.github.bumptech.glide:glide:4.12.0'
     annotationProcessor 'com.github.bumptech.glide:compiler:4.12.0'
+
+    // For complete instructions on how to enable ad ID features, please see ./Documentation/README.md
```

Comment:
let's add local href to that section itself. Example: `./Documentation/README.md#advertising-identifier`

Reply:
Updated!